revelations/client/src/userInterface/pages/GameUtils/addTower.js

```diff
@@ -1,3 +1,4 @@
 export default function addTower(event){
-
-}
\ No newline at end of file
+        // add dispatch handler
```

Comment:
Need to update to reflect addTowerBase, that is pass a dispatch parameter and call it with the relevant event data